dockerfiles/Dockerfile.shellcheck

```diff
@@ -0,0 +1,9 @@
+FROM debian:stretch-slim
```

Comment:
Looks like there are some warnings related to https://github.com/koalaman/shellcheck/issues/785, but probably not an issue.

Could we use `debian:jessie` here?